src/app/tests/BUILD.gn

```diff
@@ -244,10 +231,10 @@ chip_test_suite("tests") {
     ":operational-state-test-srcs",
     ":ota-requestor-test-srcs",
     ":power-cluster-test-srcs",
-    ":thread-network-directory-test-srcs",
     ":time-sync-data-provider-test-srcs",
     "${chip_root}/src/app",
     "${chip_root}/src/app:attribute-persistence",
+    "${chip_root}/src/app/clusters/thread-network-directory:server",
```

Comment:
we should name the targets for cluster foo as `foo` so that the server suffix is not necessary (i.e. typical/default should have the same name as the directory, extra bits should be differently named if necessary)

Reply:
I was thinking we could have client and server in the same directory, for those where client code exists?